app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+
+  def tests_by_level(level)
+    Test.joins('JOIN tests_users ON tests_users.test_id = tests.id')
+        .where(level: level, tests_users: { user_id: self.id })
```

Comment:
```suggestion
        .where(level: level, tests_users: { user_id: id })
```
при обращении к геттеру "селф" явно не пишут